### ISSUE TITLE
5.14.4, processes function does not capture commandline properly

### ISSUE THREAD

**Issue by emmkimme** [sebhildebrandt/systeminformation#752]
**Describe the bug**
On Windows platform, the CommandLine property is corrupted

**Current Output**
here is the lines splitting from the powershell output
```
[
  "ProcessId       : 3844",
  "ParentProcessId : 10720",
  "ExecutionState  : ",
  "Caption         : electron.exe",
  "CommandLine     : C:\\kobra-local\\UniversalContainer\\dev\\tools\\node_modules\\electron\\dist\\electron.exe ",
  "                  --type=crashpad-handler --user-data-dir=C:\\Users\\U8006728\\AppData\\Roaming\\Electron /prefetch:7 ",
  "                  --no-rate-limit --monitor-self-annotation=ptype=crashpad-handler ",
  "                  \"--database=C:\\Users\\U8006728\\AppData\\Local\\Temp\\Refinitiv Workspace Crashes\" ",
  "                  --url=https://dst.platform.refinitiv.com:443/api/crashes/native --annotation=_companyName=Refinitiv ",
  "                  \"--annotation=_productName=Refinitiv Workspace\" --annotation=_version=16.2.8.3 ",
  "                  --annotation=prod=Electron --annotation=ver=16.2.8-RFV.3 ",
  "                  --initial-client-data=0x558,0x580,0x588,0x530,0x58c,0x7ff62bceb920,0x7ff62bceb930,0x7ff62bceb940",
  "ExecutablePath  : C:\\kobra-local\\UniversalContainer\\dev\\tools\\node_modules\\electron\\dist\\electron.exe",
  "UserModeTime    : 0",
  "KernelModeTime  : 1406250",
  "WorkingSetSize  : 31277056",
  "Priority        : 8",
  "PageFileUsage   : 10960",
  "CreationDate    : 2022-12-21 16:03:39",
]
```
CommandLine is dispatched on several lines and the gathering does not work so well, final property lost some critical spaces:
"C:\\kobra-local\\UniversalContainer\\dev\\tools\\node_modules\\electron\\**dist\\electron.exe--type=crashpad-handle**r --user-data-dir=C:\\Users\\U8006728\\AppData\\Roaming\\Electron /p**refetch:7--no-rate-limit** --monitor-self-annotation=ptype=crashpad-handl**er\"--datab**ase=C:\\Users\\U8006728\\AppData\\Local\\Temp\\Refinitiv Workspace Crashes\"--url=https://dst.platform.refinitiv.com:443/api/crashes/native --annotation=_companyName=Refinitiv\"--annotation=_productName=Refinitiv Workspace\" --annotation=**_version=16.2.8.3--annotation**=prod=Electron --annotation=ver=16.2.8-RFV.3--initial-client-data=0x558,0x580,0x588,0x530,0x58c,0x7ff62bceb920,0x7ff62bceb930,0x7ff62bceb940"
commandLine property for this process is then equal to ""C:\\kobra-local\\UniversalContainer\\dev\\tools\\node_modules\\electron\\dist\\electron.exe\"

and there is not way to get the executable and args properly identified.

**Expected behavior**
Expect to have the original commandline

**Environment (please complete the following information):**
 - systeminformation 5.14.4
 - OS: Windows 10
 - Hardware: Dell laptop

**Comment by xavier-deryckere** [comment]
I guess that adding a space when gathering the command line parts may do the trick?

processes.js

                      if (additionalCommand && line.toLowerCase().startsWith(' ')) {
                        // instead of this:
                        // commandLine = commandLine + line.trim();
                        // do this:
                        commandLine = commandLine + ' ' + line.trim();
                      } ...

**Comment by sebhildebrandt** [comment]
@emmkimme @xavier-deryckere ... Thank you for your discovery and proposal! I came across this already last week while fixing another issue... will provide my solution tomorrow. Hope this is Ok for you.

**Comment by sebhildebrandt** [comment]
@emmkimme @xavier-deryckere ... I had a look: actually the added space is what I did ... I now pushed this to GitHub. Can you check it if this does the trick?? Then I will make a new release ...

**Comment by emmkimme** [comment]
Hi
will do test, but I think this PR https://github.com/sebhildebrandt/systeminformation/pull/753, would solve most of the issues and is more easy to read/maintain.

**Comment by emmkimme** [comment]
it works, thanks for the quick fix

**Comment by emmkimme** [comment]
BTW, I found this code
```
            for (let i = 0; i < parts.length; i++) {
              const printer = parseWindowsPrinters(parts[i].split('\n'), i);
              if (printer.name || printer.model) {
                result.push(parseWindowsPrinters(parts[i].split('\n'), i));
==> result.push(printer); ?!
              }
            }
```

**Comment by sebhildebrandt** [comment]
@emmkimme ... oh ...

which would then be much easier like this:

```
            for (let i = 0; i < parts.length; i++) {
              const printer = parseWindowsPrinters(parts[i].split('\n'), i);
              if (printer.name || printer.model) {
                result.push(printer);
              }
            }
```

Fixed this also! Version 5.16.7 just released. Thank you for your support!!!
